utils/spacewalk-utils.changes

```diff
@@ -1,3 +1,5 @@
+- Fix Debian 11 repo name in spacewalk-common-channels
```

Comment:
BTW; adjust this changelog entry. In fact you are changing the client tools channels (don't remember now if repos also have this "human" name review it for the SQL script as well):
```suggestion
- Fix Debian 11 client tools repository name in spacewalk-common-channels
```